src/components/FallenHeroes.tsx

```diff
@@ -40,22 +42,24 @@ function FallenHeroes() {
   console.log(campaign);
   return (
     <div>
-      <input
-        type="text"
-        className="input_fallen"
-        value={fallenHero}
-        placeholder="Input your fallen hero..."
-        onChange={handleChange}
-      />
-      <button className="btn input_fallen" onClick={addFallenHero}>
-        Send to the coffin
-      </button>
-      <header className="header">
-        <button className="btn list_fallen">List fallen heroes</button>
-        <div className="fallen__heroes__panel">
-          <ul className="fallen_heroes__list">{campaign.fallenHeroes}</ul>
-        </div>
-      </header>
+      <form onSubmit={addFallenHero}>
```

Comment:
Careful with what you include in the form. See what happens when you click "List fallen heroes". Can you find out why it happens?

Reply:
I have already adjusted this. I guess it was expecting some onSubmit also on that button or so, when I included it.

Reply:
no, actually it was functioning like just another submit for the input. The submit on that button automatically inferred or so

Reply:
will finalize this during rebasing

Reply:
Correct. It happened because `type="submit"` is the default value for `button`. Always make sure a form only includes one submit button and it holds only the necessary elements.